aws/logs_monitoring/README.md

```diff
@@ -138,6 +138,10 @@ If you can't install the Forwarder using the provided CloudFormation template, y
 
 If you encounter issues upgrading to the latest version, check the Troubleshooting section.
 
+### Upgrade an older version to +3.97.0
+
+Since version 3.97.0 the Lambda function has been updated to require **Python 3.11**. If upgrading an older forwarder installation to 3.97.0 or above, ensure the AWS Lambda function is configured to use Python 3.11
```

Comment:
it seems from most other files that we actually support 3.10 and 3.11 which is not really clear in this doc

Reply:
we can update the readme to include both versions, I followed the convention in the readme file